[kube-state-metrics] Race condition in GitHub workflow

### Describe the bug a clear and concise description of what the bug is.

When merging PR https://github.com/prometheus-community/helm-charts/pull/2652, the GitHub workflow failed with the error:

```
Error: signing [ghcr.io/prometheus-community/charts/kube-state-metrics:state-metrics-4.22.2]: accessing entity: entity not found in registry
main.go:62: error during command execution: signing [ghcr.io/prometheus-community/charts/kube-state-metrics:state-metrics-4.22.2]: accessing entity: entity not found in registry
```

Didn't had time to check but this could be a race condition that could have been introduced in PR https://github.com/prometheus-community/helm-charts/pull/2631.

Failed attempt : https://github.com/prometheus-community/helm-charts/actions/runs/3414205456/jobs/5681851342

Worked on re-run.

@scottrigby @monotek 

### What's your helm version?

N/A

### What's your kubectl version?

N/A

### Which chart?

kube-state-metrics

### What's the chart version?

4.22.2

### What happened?

_No response_

### What you expected to happen?

_No response_

### How to reproduce it?

_No response_

### Enter the changed values of values.yaml?

_No response_

### Enter the command that you execute and failing/misfunctioning.

Failed attempt: https://github.com/prometheus-community/helm-charts/actions/runs/3414205456/jobs/5681851342

### Anything else we need to know?

_No response_

--- a/charts/kube-state-metrics/Chart.yaml
+++ b/charts/kube-state-metrics/Chart.yaml
@@ -7,7 +7,7 @@ keywords:
 - prometheus
 - kubernetes
 type: application
-version: 4.22.2
+version: 4.22.3
 appVersion: 2.6.0
 home: https://github.com/kubernetes/kube-state-metrics/
 sources:
@@ -18,4 +18,4 @@ maintainers:
 - name: mrueg
   email: manuel@rueg.eu
 - name: dotdc
-  email: davidcalvertfr@gmail.com
+  email: david@0xdc.me

